Sources/JavaScriptKit/JSObject.swift

```diff
@@ -32,6 +32,16 @@ public class JSObjectRef: Equatable {
         getJSValue(this: self, index: Int32(index))
     }
 
+    public func instanceof(_ constructor: JSValue) -> Bool {
+        instanceof(constructor.function!)
```

Comment:
This library shouldn't provide unsafe method

Reply:
I changed it to default to `false` if you don’t pass a function in. Does that make sense?

Reply:
I think this convenience method is unnecessary. The users should handle the non-function case.

Reply:
I agree, `new` is similarly defined on `JSFunctionRef`, not `JSObjectRef` IIRC